packages/api-symphony-compatibility/src/mapping.ts

```diff
@@ -0,0 +1,56 @@
+const containerjsSsf = ssf;
+
+namespace map {
+  export class ssf {
+    static activate(windowName: string) {
+      // Could use window.focus in containerjs, but no way of selecting window by name yet
```

Comment:
Not using ContainerJS. The only method we have is `getCurrentWindow`, but it should be relatively easy to add this in.